buildpack-run.sh

```diff
@@ -1,2 +1,6 @@
 # This is buildpack-run.sh
 rm -rf .git && rm -rf build/ && git clone https://github.com/avaire/avaire.git && cp -r avaire/.git . && cp src/main/resources/config.yml . && rm -rf avaire/ && echo script worked.
+if [ -d "src/main/resources/plugins" ]; then
```

Comment:
Will discuss this with chaNcharge, however if i look at how he designed it, i would say he does the following:
He checks if the users, if he made a fork and added his plugins there: `src/main/resources/plugins`. That it will copy if something is in there, to the top directory. This is where AvaIre.jar, config.yml are.

I'll discuss if we can make this a bit easier to understand.